elastic/kibana
# [Infra] Use asset details flyout overview and metadata for containers

Closes #183727
## Summary

This PR adds asset details flyout to the inventory containers view. The feature flag `observability:enableInfrastructureContainerAssetView` should be enabled to see it.
To handle the asset type switching in the asset details flyout this PR adds asset type to the `assetDetailsFlyout` URL param 

Before:
![image](https://github.com/elastic/kibana/assets/14139027/3b4fe8a1-d874-46a4-801f-2e0f91880034)

After: 
- Overview
![image](https://github.com/elastic/kibana/assets/14139027/df54cc5c-f80f-40c8-9189-9d390e2c4eff)
- Metadata
![image](https://github.com/elastic/kibana/assets/14139027/cd4a5ba9-33fc-428a-bb0f-571d820260fc)

## Testing 
- Enable `observability:enableInfrastructureContainerAssetView` in infra settings 
- Go to containers view, click on a container, and check the content

https://github.com/elastic/kibana/assets/14139027/5ba578e6-6cdd-4561-a1f6-27ff4962ef6f
